contracts/contracts/coordination/Coordinator.sol

```diff
@@ -18,11 +13,11 @@ interface ApplicationInterface {
 contract Coordinator is Ownable {
 
     // Ritual
-    event StartRitual(uint32 indexed ritualId, address indexed initiator, address[] nodes);
+    event StartRitual(uint32 indexed ritualId, address indexed initiator, address[] participants);
     event StartTranscriptRound(uint32 indexed ritualId);
```

Comment:
Not necessarily related to this PR, but @KPrasch and I were thinking that these two events could be combined into one. It's just superfluous to have the two of them. 

```
        emit StartRitual(id, msg.sender, providers);
        emit StartTranscriptRound(id);
```

Reply:
It's a possibility, but I see those events to be different because they have a different audience: `StartTranscriptRound` is almost exclusively for nodes (and perhaps monitoring tools), and `StartRitual` is to be consumed by adopters, nodes and external observers.

Reply:
Sure, but there is overlap, and I'm not sure how much value just `ritual id` by itself provides.

`StartRitual` is the richer one since the `providers` value is included in it. This is more valuable than `StartTranscriptRound`, because nodes can easily check whether they are part of the ritual's cohort or not directly from the event. Getting an additional `StartTranscriptRound` in addition to `StartRitual` event doesn't add any real value.

Initially, when we mused about a check-in period between ritual initialization but before posting transcripts, it was valuable,  but I'm not so convinced anymore since we don't do any check-ins.